examples/preprocessing/plot_all_scaling.py

```diff
@@ -0,0 +1,162 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""
+=============================================================
+Compare the effect of different scalers on data with outliers
+=============================================================
+
+The feature 0 and feature 5 of california housing dataset are outside
+of the typical range [0, 1] and contain large outliers. These two
+characteristics lead to difficulties to visualize the data and, more
+importantly, they can degrade the fitting procedure of most of machine
+learning algorithms.
+
+Indeed, data spread in the standard range [0, 1] is a requirement for
+a large number of machine learning algorithms such as metrics-based
+algorithms or algorithms using gradient-based optimization.
+
+This example uses different scalers, transformers and normalizers to
+bring the data within a smaller range. Scalers are linear
+transformations and differ from each other depending of the subset of
+data which is considered to define the estimate using during
+scaling. ``QuantileTransformer`` provides a non-linear transformation
```

Comment:
"Scalers are linear (or more exactly affine) transformations and differ from each other in the way to estimate the parameters used to shift and scale each feature."